zebrad/src/application.rs

```diff
@@ -95,7 +109,7 @@ impl Application for ZebradApp {
             let tracing = self.tracing_component(command);
             Ok(vec![Box::new(terminal), Box::new(tracing)])
         } else {
-            init_tracing_backup();
+            crate::components::tracing::init_backup();
```

Comment:
Question: why do we need a tracing instance for the help, usage, revhex, and generate commands?

Reply:
@teor2345 if any errors are encountered this makes sure we still get SpanTraces in our panics / error reports.

Reply:
That's fine, as long as the default log level for these commands is warn.